packages/eslint-config-angular/template.js

```diff
@@ -68,7 +70,13 @@ module.exports = {
             and new applications in the chance that they'll need to be localized in the future. Disable this rule
             if an application will never be localized.
         */
-        '@angular-eslint/template/i18n': ['error', { checkId: false }],
+        '@angular-eslint/template/i18n': [
+            'error',
+            {
+                checkId: false,
+                ignoreAttributes: [...ignoreAttributes.nimble, ...ignoreAttributes['systemlink-lib-angular'], ...ignoreAttributes.jqx, ...ignoreAttributes.material]
```

Comment:
If apps choose to augment the list and we add more categories seems like it would be easy to get out of sync.

Speculating but I think it's unlikely folks will be really concerned about removing things from the list. But having the full list easily available to easily append to could be useful. Should we have that as an export and use it here?

Reply:
I've moved the attribute lists into a const to allow adding an `all` option to the module exports that concats all 4 together, and used that `all` option here.